### DISCUSSION TITLE
In README.md add a small instruction in Documentation dot

### DISCUSSION THREAD

**Discussion by Magicianred** [ptkdev/all-shields-cli#11] [General]
In README.md, Documentation dot, add a note to install ```markserv``` in global mode.

**Comment by ptkdev** [comment]
Nope, markserv is available in [package.json](https://github.com/ptkdev/all-shields-cli/blob/be3d4b3315292d0c1d0fc290f6147567f11e8eec/package.json#L77), you can run `npm run docs` without install markserv globally or use `npx markserv`

**Reply by Magicianred** [reply]
You are right, it's in package.json, so I'll have made some mistakes :)
